frontend/src/pages/org/crawl-detail.ts

```diff
@@ -852,12 +852,13 @@ export class CrawlDetail extends LiteElement {
     ];
 
     return html`
-      <div class="text-center">
-        <sl-radio-group value=${this.crawl.scale}>
+      <div>
+        <sl-radio-group value=${this.crawl.scale} help-text="Increasing parallel crawler instances can speed up crawls, but may increase the chances of getting rate limited.">
```

Comment:
We wrap text strings in the [Lit `msg` function](https://lit.dev/docs/localization/overview/#making-strings-and-templates-localizable) to make them localizable:
```suggestion
        <sl-radio-group value=${this.crawl.scale} help-text=${msg("Increasing parallel crawler instances can speed up crawls, but may increase the chances of getting rate limited.")}>
```
The `${}` wrapped around `msg()` allows you to use javascript in the HTML markup.